tests/PySys/mqtt_port/mqtt_port_change_connection_works/run.py

```diff
@@ -0,0 +1,115 @@
+from environment_c8y import EnvironmentC8y
+import sys
+import time
+
+sys.path.append("environments")
+
+"""
+Validate changing the mqtt port using the tedge command
+
+Given a configured system, that is configured with certificate created and registered in a cloud
+When the thin edge device is disconnected from cloud, sudo tedge disconnect c8y/az
+When mqtt.port is set using tedge with sudo
+When the thin edge device is connected to cloud, sudo tedge connect c8y/az
+Now validate the services that use the mqtt port
+   Validate tedge config set mqtt.port
+   Validate tedge mqtt pub/sub
+   Validate tedge connect c8y/az --test
+   Validate tedge_mapper status
+
+"""
+
+
+class MqttPortChangeConnectionWorks(EnvironmentC8y):
```

Comment:
Depending on what else you are using from the environment you can think of avoiding in. When you just need something like EnvironmentC8y that does not connect more than one we can could add an environment for that

Reply:
I was just thinking about setting up the tenant ID and password, but these will be set by default when the system tests starts. So, I feel I do not need this Environment at all.